webapp/src/components/cardDetail/cardDetailContents.test.tsx

```diff
@@ -0,0 +1,258 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react'
+
+import {fireEvent, render} from '@testing-library/react'
+
+import {IntlProvider} from 'react-intl'
+import {act} from 'react-dom/test-utils'
+
+import {DndProvider} from 'react-dnd'
+import {HTML5Backend} from 'react-dnd-html5-backend'
+
+import {TestBlockFactory} from '../../test/testBlockFactory'
+
+import CardDetailContents from './cardDetailContents'
+
+const wrapProviders = (children: any) => {
+    return (
+        <DndProvider backend={HTML5Backend}>
+            <IntlProvider locale='en'>{children}</IntlProvider>
+        </DndProvider>
+    )
+}
+
+global.fetch = jest.fn()
+
+document.createRange = () => {
+    const range = new Range()
+
+    range.getBoundingClientRect = jest.fn()
+
+    range.getClientRects = () => {
+        return {
+            item: () => null,
+            length: 0,
+            [Symbol.iterator]: jest.fn(),
+        }
+    }
+    return range
+}
+window.focus = jest.fn()
```

Comment:
Should this code be replaced with calling [recently added mockDOM](https://github.com/mattermost/focalboard/blob/main/webapp/src/testUtils.tsx#L8-L25)?